src/game_ui.c

```diff
@@ -1998,9 +1998,9 @@ void hud_bananas(Object_Racer *racer, s32 updateRate) {
         }
         if (prevSprite == 0) {
             sprite_opaque(TRUE);
-            set_viewport_tv_type(OS_TV_TYPE_NTSC);
+            set_viewport_tv_type(TRUE);
```

Comment:
`set_viewport_tv_type` definitely needs a rename. Maybe `enable_pal_viewport_height_adjust` or something like that?